roles/openmrs/tasks/main.yml

```diff
@@ -94,6 +94,7 @@
   become_user: root
   get_url:
     url: "https://netcologne.dl.sourceforge.net/project/openmrs/releases/OpenMRS_Platform_{{ openmrs_version }}/openmrs.war"
+    validate_certs: false
```

Comment:
@dubdabasoduba Could this be a variable? Default it to `true`. Since it seems like an exception, set it to false for deployments that don't want the certificates to be validated.